src/queue.rs

```diff
@@ -0,0 +1,102 @@
+//! The queue of nodes that keeps track of pending operations.
+
+use crate::node::Node;
+use crate::sync::atomic::{AtomicPtr, Ordering};
+
+use crossbeam_utils::CachePadded;
```

Comment:
`concurrent-queue` explicitly uses `cache-padded` for this instead of `crossbeam-utils`, this crate should do the same as it is usually used together with `concurrent-queue`

Reply:
On the current `master` branch of `concurrent-queue`, it's been ported to use `crossbeam-utils`.

https://github.com/smol-rs/concurrent-queue/blob/master/src/unbounded.rs

Reply:
ah ok.